File: documentation/user/en/use/api/example/manual-transaction-management.cs
// open session manually
EvitaClientSession session = evita.CreateReadWriteSession("evita");
// initiate session manually
long txId = session.OpenTransaction();
// in case of error, mark transaction as rollback only
session.SetRollbackOnly();
// close the transaction
session.CloseTransaction();
// or close the entire session
// this implicitly closes the internal transaction
session.Close();

File: documentation/user/en/use/api/example/manual-transaction-management.java
final EvitaSessionContract session;
try{
	// open session manually, read-write sessions automatically open a transaction
	session = evita.createReadWriteSession("evita");
	// in case of error, mark transaction as rollback only
	session.setRollbackOnly();
} finally {
	try{
		// close the entire session - this implicitly closes the internal transaction
		// if `setRollbackOnly` wouldn't have been called, the transaction would have been committed
		session.close();
	} catch (RollbackException e){
		// handle information about rollback
	}
}
